src/python/dbs/apis/dbsClient.py

```diff
@@ -400,7 +401,16 @@ def __callServer(self, method="", params={}, data={}, callmethod='GET', content=
                 print("HTTP={} URL={} method={} params={} data={} headers={}".format(callmethod, self.url, method, params, data, request_headers))
             self.http_response = method_func(self.url, method, params, data, request_headers)
         except HTTPError as http_error:
-            self.__parseForException(http_error)
+            if http_error.code == 200:
+                self.__parseForException(http_error)
+            else:
+                print("### HTTPError ###")
```

Comment:
Prints are for human, not program. You may want to hide these print in debug mode.

Reply:
No, I doubt it, the user should be able to understand the error. If error hides the details of HTTP response I think it is much better to see them in non-verbose mode.

Reply:
I realized you like to print out everything in the code and it is OK. but why you think "raise http_error" will not provide enough error info to the clients? In the end, the errors are handled by program that will not look into what you print out, but the exceptions. If the user really want to see the print out, they should turn on debug. 
In addition to this, the huge log files w/ unnecessary info will make it difficult for one to find useful content under emergency debugging . This is my two cents.

Reply:
Yuyi, I raise because it is already an exception. I print because debug info is too late when exception happen when user do not use debug mode. If exception happened we need to understand what happen. As such the more information user have the better chances to understand what happen. Otherwise we come back to square one that exception is cryptic and users have no clue to understand what went wrong.